source/draw/RasterMonochrome.ooc

```diff
@@ -88,7 +88,8 @@ RasterMonochrome: class extends RasterPacked {
 			converted referenceCount decrease()
 		} else {
 			for (y in 0 .. this size y)
-				for (x in 0 .. this size x) {
+				for (index in y % 2 .. this size x / 2) {
+					x := index * 2
```

Comment:
This code will not give a checker pattern since the initial offset is multiplied by two. Add two to the counter instead and let it end below size x as usual.